index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+  <head><link rel="stylesheet" href="styles.css"></head>
+  <body>
+    
+    <h1 id="title">freeCodeCamp Survey Form</h1>
+    <p id="description">Thank you for taking the time to help us improve the platform</p>
```

Comment:
it's a good practice to enclose h1 and p tags inside &lt;div&gt; tags

Reply:
Why it's a good practice to use &lt;div&gt; tags because

The &lt;div&gt; tag defines a division or a section in an HTML document.

The &lt;div&gt; tag is used as a container for HTML elements - which is then styled with CSS or manipulated with JavaScript.

The &lt;div&gt; tag is easily styled by using the class or id attribute.

Any sort of content can be put inside the &lt;div&gt; tag!